### PR TITLE
Wip frameborder rewrite mb

### PR DESCRIPTION
**Pull request by maxwellbenton**
@sgharms after reviewing iframes a bit more I rewrote some of the edited content on wip-sgh-update-attribute-copy, if you don't mind taking a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,22 +48,27 @@ there as well.
...
-is consider _deprecated_ meaning "likely to be removed from the standard."
+off.  
 
-We see this appear often in `iframe` links as providers try to support _as many
-browsers as possible_. It's important to understand what it means, but you
-probably won't need to write it much.  If you do, you'll want to set both
-`frameborder="0"` AND `style="border:0"`.
+The `frameborder` attribute is considered _deprecated_, meaning "likely to be
+removed from the standard." In modern browsers, we can control borders _just_
+using CSS, as with our example, `style="border:0"`.
```
Comment (sgharms) [inline README.md]: At this point they haven't seen CSS yet. Could we add something like "you might not be familiar with CSS yet. For the time being feel free to simply add this attribute"

Could you work with that idea a bit? Basically don't assume they _know_ css at this point.

File: README.md [hunk 2]
```diff
@@ -48,22 +48,27 @@ there as well.
...
 
 Like the `required` attribute for HTML form elements, `allowfullscreen` is built
-into HTML5. If the provider provides support for a JavaScript method called
-`requestFullScreen()`, then your page can send the `iframe` full screen. If
-not, it can't. It's important to be able to understand this attribute and
-realize that you can't get around any restrictions unless the _provider_ wants
-you to. For more details visit [MDN][mdn-allowfull].
+into HTML5. The `allowfullscreen` attribute uses a JavaScript method called
+`requestFullScreen()` to send the `iframe` to full screen. If `allowfullscreen`
+isn't included, the element can't be placed into full screen mode.. It's
```
Comment (sgharms) [inline README.md]: extra `.`

File: README.md [hunk 3]
```diff
@@ -48,22 +48,27 @@ there as well.
...
+important to be able to understand this attribute and realize that you can't get
+around any restrictions unless the _provider_ wants you to. For more details
+visit [MDN][mdn-allowfull].
 
 [mdn-allow]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe
```
Comment (sgharms) [inline README.md]: Could you fix this to match the use `[mdn-allowfull]` is a better name.

**Review by sgharms — CHANGES_REQUESTED** [review]

**Comment by maxwellbenton** [comment]
Made changes requested based on sgharms review. Merging to wip-sgh-update-attribute-copy then to master

**Pull request merged**